app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -462,8 +462,12 @@ fun pictrsImageThumbnail(src: String, thumbnailSize: Int): String {
     return "$host/pictrs/image/$path?thumbnail=$thumbnailSize&format=webp"
 }
 
-fun isImage(url: String): Boolean {
-    return imageRegex.matches(url)
+fun isImage(url: String?): Boolean {
```

Comment:
Why was this changed?

Reply:
Oh thanks for pointing that out. This belongs to #877 but is also wrong: 
I thought I could get away with one less check for null statement and I wrongly assumed `someRegex.matches(null)` results in `false` but it actually leads to a NullPointerException.

Anyway, doesn't belong in this PR anyway and will be removed.